templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +25,12 @@
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                  <a href="{% url "taxi:manufacturer-update" pk=manufacturer.id %}">Update</a>
```

Comment:
Ensure that the `manufacturer-update` URL is correctly defined in your `urls.py` file to avoid any broken links.